### ISSUE TITLE
DETECT: install IBEX

### ISSUE THREAD

**Issue by ChrisM-S** [ISISComputingGroup/IBEX#4260]
As a member of the detectors group I would like an ibex installation available and configured on the two detector testing machines (which are used for testing neutron detectors before installation onto instruments).

The machines currently run with old SECI in 1500MB per machines but are being moved to a newer host server with at least 16GB RAM (a decommissioned NDHEVS). I would like to use python for some specialist testing scripts and run both machines on IBEX.  They both talk to a DAE and BEGIN and END runs and store data to the ISIS data archive.

- [x] machines have 30GB free disk space
- [x] machines have 8GB ram
- [ ] IBEX is installed
- [x] Components created for:
    * No sample environment
contact Davide Raspino
- [ ] both machines added to appropriate nagios groups
- [ ] both machines added to instrument list

**Comment by kjwoodsISIS** [comment]
Not done in SPRINT_2019_04_25.  Remove from sprint.

**Comment by kjwoodsISIS** [comment]
Any sign of this work being required on a specific timescale?

**Comment by ChrisM-S** [comment]
This migration is becoming more urgent because testing for Detectors and ADCs for the LOKI instrument (in kind project) done on TS2 instruments is done on IBEX instruments and Genie Python scripting is needed on at least one of DETECT1 or DETECT2.

They expect to be testing from January at some point but would be happy with just DETECT1 converted to IBEX and keeping DETECT2 on SECI for the time being (so no need to support both systems concurrently on the same VM).

Being that the plan for this was to use the new Windows 10 clone which already runs IBEX, the only significant question is gap analysis of what we would need to add for the ICP/DAE and data collection to function in this system.  If `\Data` is cloned to the new system from an existing instrument is the rest of the data collection system potentially functional with an up to date DAE installation on the Apps disk? (could be linked to `\LabVIEW Modules\DAE` for path equivalence)

**Comment by ChrisM-S** [comment]
I've just had a co-incidental request from the scientist to ask if we are able to do this remotely so that they are able to use at least one DETECT machine on IBEX when lockdown is eased.

Maybe we should move this into High? the reason being that they can potentially test detectors as soon as they can get anyone back on site (and this is part of our delivery for the LOKI instrument on ESS).  It's not necessary to wait until the next ISIS cycle for work to be done.

We are still dependent on #4415 for this so that would obviously be higher.

**Comment by ChrisM-S** [comment]
Davide has confirmed that the DAEs were both left on, so as long as we are careful, lack of anyone with access to the area should not be an issue (fingers crossed we don't have any power glitches).  The DAEs connections are also interchangeable without physical access so we would have still have a spare if one locks up.

**Comment by DominicOram** [comment]
@ChrisM-S to ask for priority from detector group given COVID-19 issues

**Comment by ChrisM-S** [comment]
It's a moving target, but currently they have what they need, I'll close as anything further would be another ticket
